pyfv3/_config.py

```diff
@@ -195,86 +195,60 @@ class DynamicalCoreConfig:
     vtdm4: float = DEFAULT_FLOAT
     z_tracer: bool = DEFAULT_BOOL
     do_qa: bool = DEFAULT_BOOL
-    layout: Tuple[int, int] = NamelistDefaults.layout
-    grid_type: int = NamelistDefaults.grid_type
-    u_max: float = NamelistDefaults.u_max  # max windspeed for dp config
-    do_f3d: bool = NamelistDefaults.do_f3d
-    inline_q: bool = NamelistDefaults.inline_q
-    do_skeb: bool = NamelistDefaults.do_skeb  # save dissipation estimate
-    use_logp: bool = NamelistDefaults.use_logp
-    moist_phys: bool = NamelistDefaults.moist_phys
-    check_negative: bool = NamelistDefaults.check_negative
+    layout: Tuple[int, int] = (1, 1)
+    grid_type: int = 0
+    u_max: float = 350.0  # max windspeed for dp config
```

Comment:
If we change it to

```suggestion
    u_max: float = 350.0
    """max windspeed for dp config"""
```

then it automatically starts appearing in the [auto-generated docs](https://noaa-gfdl.github.io/PyFV3/top/_config/). This is somewhat tangential. I wouldn't do all, but maybe the ones that anyway get touched.

Reply:
Thanks! I've updated the comments that got touched.